bench/micro/Bench/Database/LSMTree/Internal/Merge.hs

```diff
@@ -201,28 +201,28 @@ benchMerge conf@Config{name} =
             --    thread `runs` through the environment, too.
             -- 2. It forces the result to normal form, which would traverse the
             --    whole run, so we force to WHNF ourselves and just return `()`.
+
+            -- We make sure to immediately close resulting runs so we don't run
+            -- out of file handles or disk space. However, we don't want it to
+            -- be part of the measurement, as it includes deleting files.
+            -- Therefore, ... TODO
```

Comment:
There is a dangling TODO here

Reply:
Oops, that slipped through. Thanks for pointing it out. https://github.com/IntersectMBO/lsm-tree/pull/547